gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/OrchestratorTest.java

```diff
@@ -86,6 +92,8 @@ public class OrchestratorTest {
   private FlowSpec flowSpec;
   private ITestMetastoreDatabase testMetastoreDatabase;
   private Orchestrator dagMgrNotFlowLaunchHandlerBasedOrchestrator;
```

Comment:
seems misnomer, as we no longer have any `Orchestrator` capable of using the DagMgr (now completely removed) instead of `FlowLaunchHandler`.  suggest to rename merely to `orchestrator`

Reply:
updated to orchestrator